sa/rocsp_sa.go

```diff
@@ -0,0 +1,35 @@
+package sa
+
+import (
+	"context"
+	"time"
+
+	rocsp_config "github.com/letsencrypt/boulder/rocsp/config"
+)
+
+type RocspWriteSource interface {
```

Comment:
Why is this named "Source"? I would expect "RocspWriter". Also, does this need to be exported outside SA?